docs/src/gettingstarted.md

```diff
@@ -0,0 +1,222 @@
+and following the instructions for your platform.
+`BiobakeryUtils.jl` should work on any julia version >= 1.6.0.
+
+Alternatively, you can you [`jill.py`](https://github.com/johnnychen94/jill.py),
+which is an easy-to-use python utility for installing julia.
+
+### Launching julia from the terminal
+
+If you download the "app" versions of julia from the downloads page above,
+you may also want to add `julia` to your shell's `$PATH`
+so that you can launch it from your terminal.
+For windows users, you can look [look here](https://julialang.org/downloads/platform/#adding_julia_to_path_on_windows_10)
+for instructions.
+Mac users, [see here](https://julialang.org/downloads/platform/#optional_add_julia_to_path)
+for instructions.
+
+## Making a project
+
+In julia, it's typically a good idea to use ["projects"](https://pkgdocs.julialang.org/v1/environments/)
+to organize your package dependencies
+(this is similar to "environments" that `conda` uses).
+
+To do this, make a directory and "activate" it in the julia Pkg REPL.
+
+```sh
+$ mkdir my_project
+
+$ cd my_project
+
+$ julia
+
+               _
+   _       _ _(_)_     |  Documentation: https://docs.julialang.org
+  (_)     | (_) (_)    |
+   _ _   _| |_  __ _   |  Type "?" for help, "]?" for Pkg help.
+  | | | | | | |/ _` |  |
+  | | |_| | | | (_| |  |  Version 1.6.1 (2021-04-23)
+ _/ |\__'_|_|_|\__'_|  |  Official https://julialang.org/ release
+|__/                   |
+
+julia> # press ] to enter the Pkg REPL
+
+(@v1.6) pkg> activate .
+  Activating new environment at `~/my_project/Project.toml`
+
+(my_project) pkg> # press backspace to get back to julia REPL
+
+julia>
+```
+
+[![asciicast](https://asciinema.org/a/440135.svg)](https://asciinema.org/a/440135)
+
+So far, this is still just an empty directory,
+but you can also use the Pkg REPL to install packages, like `BiobakeryUtils.jl`.
+
+```sh
+(my_project) pkg> add BiobakeryUtils
+```
+
+[![asciicast](https://asciinema.org/a/8vMgAdlGV63VztAGhUAlGQ5ai.svg)](https://asciinema.org/a/8vMgAdlGV63VztAGhUAlGQ5ai)
+
+Once this process completes, the directory will now contain a `Project.toml` file
+that contains `BiobakeryUtils.jl` as a dependency,
+and a `Manifest.toml` file that contains all of the exact info about
+dependencies installed for this environment.
+
+In the future, you can launch julia with the environment already activated
+using `julia --project` if your working directory is `my_project/`,
+or `julia --project=<path to project>` if you're in a different working directory
+(eg. `julia --project=~/my_project` if `my_project/` is in the home directory).
+
+## Using `bioBakery` command line tools
+
+Some functions provided by this package (eg [`humann_regroup`](@ref) and [`humann_rename`](@ref)),
+require the appropriate `bioBakery` tools to be installed and accessible from the julia `shell` environment.
+The easiest way to do this is to use `Conda.jl`,
```

Comment:
[`Conda.jl`](https://github.com/JuliaPy/Conda.jl)

Reply:
fixed in #69